OracleWebCenterContent/dockerfiles/README.MD

```diff
@@ -0,0 +1,315 @@
+Oracle WebCenter Content on Docker
+==================================
+
+## Contents
+
+### 1. [Introduction](#1-introduction-1)
+### 2. [Hardware and Software Requirements](#2-hardware-and-software-requirements-1)
+### 3. [Pre-requisites](#3-pre-requisites-1)
+### 4. [Building Oracle WebCenter Content Docker Image](#4-building-oracle-webcenter-content-docker-image-1)
+### 5. [Running Oracle WebCenter Content Docker Container](#5-running-oracle-webcenter-content-docker-container-1)
+### 6. [License](#6-license-1)
+### 7. [Copyright](#7-copyright-1)
+# 1. Introduction
+This project offers scripts to build an Oracle WebCenter Content docker image based on 12.2.1.4.0 release. Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
```

Comment:
Please replace the first instance of "docker" with "container" and remove the second instance completely.